torch/testing/_internal/common_utils.py

```diff
@@ -300,9 +300,11 @@ def numpy_dtype(dtype):
     assert TEST_NUMPY
 
     torch_to_numpy_dtype = {
-        torch.half   : numpy.float16,
-        torch.float  : numpy.float32,
-        torch.double : numpy.float64
+        torch.half       : numpy.float16,
+        torch.float      : numpy.float32,
+        torch.double     : numpy.float64,
+        torch.complex64  : numpy.complex64,
```

Comment:
what is this change for?

Reply:
It lets the new test map torch dtypes to the corresponding NumPy dtype so it can compare behavior with NumPy.